packages/react-ui-core/src/Ratings/RatingBar.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import RatingItem from './RatingItem'
+
+export default class RatingBar extends Component {
+  static propTypes= {
+    theme: PropTypes.object,
+  }
+  static defaultProps = {
+    theme: {},
+  }
+
+  render() {
+    const { max, rating, backgroundColor, total, theme } = this.props
+    const ratingItems = []
+    let currentRating = rating
+    while (ratingItems.length < max) {
```

Comment:
What do you think about doing something a little less convoluted like:
```
 get ratingItems() {
    const {
      max,
      rating,
      backgroundColor,
      theme
    } = this.props

    return [...Array(max).keys()].map(index => (
      <RatingItem
        width={`${(rating - index) * 100}%`}
        index={index}
        backgroundColor={backgroundColor}
        theme={theme}
      />
    ))
  }
```

render method would look like this then:
```
<div className={theme.ratingcomp}>{this.ratingItems}</div>
```